cargo-dist/src/generic_build.rs

```diff
@@ -132,3 +148,37 @@ pub fn build_generic_target(dist_graph: &DistGraph, target: &GenericBuildStep) -
 
     Ok(())
 }
+
+/// Similar to the above, but with slightly different signatures since
+/// it's not based around axoproject-identified binaries
+pub fn run_extra_artifacts_build(dist_graph: &DistGraph, target: &ExtraBuildStep) -> Result<()> {
+    eprintln!(
+        "building extra artifacts target (via {})",
+        target.build_command.join(" ")
+    );
+
+    let result = run_build(dist_graph, &target.build_command, None)?;
+    let dest = dist_graph.dist_dir.to_owned();
+
+    if !result.status.success() {
+        println!("Build exited non-zero: {}", result.status);
+    }
+    eprintln!();
+    eprintln!("stdout:");
+    stderr().write_all(&result.stdout).into_diagnostic()?;
```

Comment:
this is a weird juxtaposition... but ok i think i see why

Reply:
This is for consistency with the cargo builds, where we have things which consume the underlying build's stdout. I'd run into cases where allowing the output to go to stdout caused other parts of our build process to fail. (This code is actually from the original generic builds code, just duplicated here.)